src/app.ts

```diff
@@ -59,6 +60,23 @@ async function routes(fastify: FastifyInstance) {
   }
 }
 
+export const validatorCompiler: FastifySchemaCompiler<ZodAny> =
+  ({ schema }) =>
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (data): any => {
+    try {
+      return { value: schema.parse(data) };
```

Comment:
use [safeparse](https://zod.dev/?id=safeparse) instead of `parse` to avoid  try-catch

Reply:
Updated at 56a7efd